### PR TITLE
Karam dosa

### PR DESCRIPTION
**Pull request by kaminenileela**
please review

### CODE REVIEW

**Comment by vamsitron** [comment]
why did you add oil to karam dosa? already oil is there for plain dosa. why did you add it specifically?

**Comment by kaminenileela** [comment]
> why did you add oil to karam dosa? already oil is there for plain dosa. why did you add it specifically?
extra oil is good for karam dosa

File: recipes/dosa.txt [hunk 1]
```diff
@@ -1 +1,3 @@
 dosa flour
+karam
+oil
```
Comment (vamsitron) [inline recipes/dosa.txt:3]: why is oil added at the end?

Comment (kaminenileela) [reply recipes/dosa.txt:3]: i added at the end, so it may be tasty.

**Review by vamsitron — APPROVED** [review]

**Pull request merged**